shared/charts/gpii-preferences/templates/service_entry_googleapis.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.enableStackdriverTrace }}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "preferences.name" . }}-googleapis
+spec:
+  hosts:
+  - www.googleapis.com
```

Comment:
Same as above, why is `www.googleapis.com` needed & might make sense to merge in one `SE`.